### PR TITLE
Release v1.0.2

### PR DESCRIPTION
**Pull request by egabancho**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,11 +1,16 @@
 ..
-    Copyright (C) 2018, 2019 Esteban J. G. Gabancho.
+    Copyright (C) 2018, 2019, 2020 Esteban J. G. Gabancho.
     Invenio-S3 is free software; you can redistribute it and/or modify it
     under the terms of the MIT License; see LICENSE file for more details.
 
 Changes
 =======
 
+Version 1.0.0 (released 2020-02-17)
```
Comment (ppanero) [inline CHANGES.rst]:
```suggestion
Version 1.0.2 (released 2020-02-17)
```

File: invenio_s3/__init__.py [hunk 2]
```diff
@@ -18,9 +18,9 @@
 
 When creating a new location which will use the S3 API, the URI needs to start
 with ``s3://``, for example
-``invenio files location S3_default s3://myserver.com/b1 --default`` will
+``invenio files location S3_default s3://my-bucket --default`` will
```
Comment (ppanero) [inline invenio_s3/__init__.py]: Moderate: IIRC the name does not accept upper case (I do not understand why dashes are highlighted in the suggestions...):

```suggestion
``invenio files location s3_default s3://my-bucket --default`` will
```

Comment: As soon as this https://github.com/inveniosoftware/invenio-files-rest/pull/236 is merged, the command will be:
``` console
$ invenio files location create s3_default s3://my-bucket --default
```
Not  saying to change it now, just to be aware, for the next release.

**Comment by egabancho** [comment]
All good now, hopefully ☺️

**Comment by ppanero** [comment]
Closes https://github.com/inveniosoftware/invenio-s3/issues/13

**Comment by lnielsen** [comment]
You can remove the python 2 and 3.5 from travis and add 3.6 3.7 and 3.8. And it seems like it just one of the lower requirements that has to be bumped to pass.

**Pull request merged**